<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainScene</class>
 <widget class="QWidget" name="mainScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>开始购票</string>
   </property>
  </widget>
  <widget class="QPushButton" name="referBtn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>310</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>地图查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="existBtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>460</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
